docs/benchmarks/rabi_oscillations.ipynb

```diff
@@ -0,0 +1,495 @@
+    "\n",
+    "Now if you wanted to use `cirq.simulate` or `cirq.sample` with the parameterized circuit you would also need to specify a value for `theta`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "SMdz-yAZSwrU"
+   },
+   "outputs": [],
+   "source": [
+    "samples_at_theta_equals_2 = sim.sample(\n",
+    "    parameterized_circuit, params={theta: 2}, repetitions=10\n",
+    ")\n",
+    "samples_at_theta_equals_2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "H_H13Hc8g873"
+   },
+   "source": [
+    "You can also specify *multiple* values of `theta`, and get samples back for each value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "0zjZxGY6hIsu"
+   },
+   "outputs": [],
+   "source": [
+    "samples_at_multiple_theta = sim.sample(\n",
+    "    parameterized_circuit, params=[{theta: 0.5}, {theta: 3.141}], repetitions=10\n",
+    ")\n",
+    "samples_at_multiple_theta"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "juuWvOEphaaE"
+   },
+   "source": [
+    "Cirq has shorthand notation you can use to sweep `theta` over a range of values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "8lCb3049hqXn"
+   },
+   "outputs": [],
+   "source": [
+    "samples_at_swept_theta = sim.sample(\n",
+    "    parameterized_circuit,\n",
+    "    params=cirq.Linspace(theta, start=0, stop=3.14159, length=5),\n",
+    "    repetitions=5,\n",
+    ")\n",
+    "samples_at_swept_theta"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "wqaORMoKiAIW"
+   },
+   "source": [
+    "The result value being returned by `sim.sample` is a `pandas.DataFrame` object.\n",
+    "Pandas is a common library for working with table data in python.\n",
+    "You can use standard pandas methods to analyze and summarize your results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "bLzGV8nFiS9o"
+   },
+   "outputs": [],
+   "source": [
+    "import pandas\n",
+    "\n",
+    "big_results = sim.sample(\n",
+    "    parameterized_circuit,\n",
+    "    params=cirq.Linspace(theta, start=0, stop=3.14159, length=20),\n",
+    "    repetitions=10_000,\n",
+    ")\n",
+    "\n",
+    "# big_results is too big to look at. Plot cross tabulated data instead.\n",
+    "pandas.crosstab(big_results.theta, big_results.out).plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "b2TkL28AmBSQ"
+   },
+   "source": [
+    "## 3. The ReCirq experiment\n",
```

Comment:
Is it worth briefly explaining what a Rabi experiment is (and/or linking to somewhere for additional information)?

Reply:
Yes, but I don't have the knowledge to write that. Do you know who originally wrote this? I'll go ask them.